code/web/src/modules/survey/Survey.js

```diff
@@ -0,0 +1,157 @@
+import React, { PureComponent } from 'react'
```

Comment:
This looks great, we should decide if we want to leave this as is or if we should create different components for each row.